caster/lib/navigation.py

```diff
@@ -13,7 +13,8 @@
 from caster.lib import control
 
 
-DIRECTION_STANDARD={"sauce [E]": "up", "dunce [E]": "down", "lease [E]": "left", "Ross [E]": "right", "back": "left" }
+#DIRECTION_STANDARD={"sauce [E]": "up", "dunce [E]": "down", "lease [E]": "left", "Ross [E]": "right", "back": "left" }
+DIRECTION_STANDARD={"up": "up", "down": "down", "left": "left", "right": "right", "back": "left" }
```

Comment:
Oops sorry you can ignore that change since I was intending to keep it on local. I accidentally submitted this one from the develop branch instead of creating a feature branch. On the plus side, I didn't know about filter functions and they sound useful, so I'll definitely check them out!